docs/json/sonarr/cf/my5.json

```
@@ -0,0 +1,37 @@
+{
+  "trash_id": "9f72dc1059a6b277c21cee6a1f15f13f",
+  "trash_scores": {
+    "default": 50
+  },
+  "name": "MY5",
+  "includeCustomFormatWhenRenaming": true,
+  "specifications": [
+    {
+      "name": "MY5",
+      "implementation": "ReleaseTitleSpecification",
+      "negate": false,
+      "required": true,
+      "fields": {
+        "value": "\\b(my5)\\b[ ._-]web[ ._-]?(dl|rip)?\\b"
```

```suggestion
        "value": "\\b(MY5)\\b"
```
I think this should be enough, I highly doubt that this combination of letters and numbers is a common movie/show/episode title 😄